Add offset instructions to automation trigger (#26963)
home-assistant/home-assistant.github.io: source/_integrations/sun.markdown

<img src='/images/screenshots/more-info-dialog-sun.png' />
</p>

## Automation trigger

The sun's event listener will call the service when the sun rises or sets with
an offset.

The sun trigger need to have the type 'sun', which event (sunset or sunrise) and an optional offset.

```json
{

>     "platform": "sun",

Reviewer: This is really oldschool actually. Let's convert it to an actual YAML trigger to make it easier to understand.

    "event": "sunset",
    "offset": "-01:00:00"
}
```

| Key name | Description |
| --------- | ----------- |

> | `event` | Possible values  `sunset`,  `sunrise `

Reviewer: ```suggestion
| `event` | Possible values: `sunset` or `sunrise`
```

> | `offset` | An optional offset specified in HH:MM:SS from event trigger. <br> Positive after event is triggered. <br> Negative before event is triggered

Reviewer: - I think we should avoid "event trigger". Events in Home Assistant are a different thing; let's add context to it by saying sun trigger.

- Postive negative adds/subtracts from the time. I don't think we need to clarify that (besides, the current suggested wording was very unclear, IMHO).

```suggestion
| `offset` | An optional offset for the sun event trigger, in a positive or negative number of seconds, or specified in `HH:MM:SS` or `-HH:MM:SS`.
```

Author: It was actually positive/negative values that I struggled with, which caused me to create this PR.

Reviewer: What was the exact struggle you had with that?

Author: Trivial indeed I know that now. 
I struggled with: Does a negative number mean before or after the sun event trigger?
To me somehow it's not explicitly mentioned anywhere and it can be mistaken.
Making it explicit (like in my last commit after seeing your feedback) no one can be mistaken anymore.
